tenablesc_connector.py

```diff
@@ -549,91 +622,112 @@ def _list_policies(self, param):
             return action_result.get_status()
 
         action_result.add_data(resp_json["response"])
-        action_result.update_summary({'policy_count': len(resp_json["response"].get("usable", []))})
+        action_result.update_summary({"policy_count": len(resp_json["response"].get("usable", []))})
         message = "Total policies: {}".format(len(resp_json["response"].get("usable", [])))
 
         return action_result.set_status(phantom.APP_SUCCESS, message)
 
+    def _list_repositories(self, param):
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        ret_val, resp_json = self._make_rest_call("/repository", action_result)
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        for repository in resp_json["response"]:
+            action_result.add_data(repository)
+
+        action_result.update_summary({"repository_count": len(resp_json["response"])})
```

Comment:
Can we replace `repository_count` key with `total_repositories`? This way we can skip add the message to the action_result object. It will automatically add the message based on the summary.

Reply:
Response is like:
```
{
    "response": [{...}, ...]
}
```

So we have to manually add each repository to data

Reply:
What I meant over here was that instead of adding `repository_count` key in the summary, how about we add `total_repositories`?
```suggestion
        action_result.update_summary({"total_repositories": len(resp_json["response"])})
```

This way you can simply remove the next line and update return statement as:
`return action_result.set_status(phantom.APP_SUCCESS)`